docs/build

```diff
@@ -6,7 +6,7 @@ cd "${BASH_SOURCE%/*}"
 
 test -t 1 && OPTS='-it' || OPTS=''
 
-SPHINX_IMAGE=${SPHINX_IMAGE:-prestodev/sphinx:3}
+SPHINX_IMAGE=${SPHINX_IMAGE:-ghcr.io/trinodb/testing/sphinx:3}
```

Comment:
we should rename the image to `build/sphinx`, since it's not testing related.

this needs to be done manually (there is no actual rename option).

see also https://github.com/trinodb/docker-images/pull/82#discussion_r552990255

Reply:
Any reason for maintaining it in GHCR, can we pull their official image - https://hub.docker.com/r/sphinxdoc/sphinx

Reply:
This i do not know

Reply:
Can we fetch their official docker image and build on it ?

Reply:
I possible I would prefer that

Reply:
we already use the base image from sphinx - add a few more dependency